app/views/events/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1 class="text-center text-primary">Event</h1>
+
+<div class="container">
+  <div class="row">
+    <p><span class="font-weight-bold">Name:</span> <%= @event.name %></p>
+  </div>
+  <div class="row">
+    <p><span class="font-weight-bold">Description:</span> <%= @event.description %></p>
+  </div>
+  <div class="row">
+    <p><span class="font-weight-bold">Date:</span> <%= @event.date %></p>
+  </div>
+  <div class="row">
+    <p><span class="font-weight-bold">Creator:</span> <%= @event.creator.name %></p>
+  </div>
+
+  <div class="row justify-content-center">
+    <% if !@current_user.attended_events.exists?(@event.id) %>
+      <%= button_to "Assist", assist_event_path(event_id: @event.id), {class: 'btn btn-primary'} %>
+    <% else %>
+      <%= button_to "Assist", assist_event_path(event_id: @event.id), {class: 'btn btn-secondary', disabled: true} %>
+    <% end %>
+  </div>
+
+  <h2 class="text-center text-primary m-4">Attendees</h2>
+
+  <% @attendees.each do |a| %>
```

Comment:
- [ ] There should be no logic at all( if else, each, etc) in the views.
View logic should be inside the pertinent helper.